CalcExpr/FunctionAttributes/PreprocessAttributes/AsBooleanAttribute.cs

```diff
@@ -1,18 +1,17 @@
 ﻿using CalcExpr.Expressions;
+using CalcExpr.Expressions.Interfaces;
 
 namespace CalcExpr.FunctionAttributes.PreprocessAttributes;
 
 public class AsBooleanAttribute : PreprocessAttribute
 {
     public override IExpression Preprocess(IExpression expression)
```

Comment:
Could use the ILogicalConvertible.ConvertToLogical function instead of if statements